src/Umbraco.Web.UI/config/serilog.config

```diff
@@ -2,13 +2,36 @@
 <configuration>
     <appSettings>
 
-        <!-- Used to toggle the loge levels for the main Umbraco log files -->
+        <!-- Used to toggle the log levels for the main Umbraco log files -->
         <!-- Found at /app_data/logs/ -->
         <!-- NOTE: Changing this will also flow down into serilog.user.config -->
         <!-- VALID Values: Verbose, Debug, Information, Warning, Error, Fatal -->
         <add key="serilog:minimum-level" value="Information" />
 
-        <!-- To write to new log locations (aka Sinks) such as your own .txt files, ELMAH.io, Elastic, SEQ -->
+        <!-- NOTE: Only one logger below can be enabled, you cannot log JSON & TXT files at the same time -->
+        
+        <!-- Default JSON log file -->
+        <!-- This is used by the defualt log viewer in the Umbraco backoffice -->
+        <add key="serilog:using:File" value="Serilog.Sinks.File" />
+        <add key="serilog:write-to:File.formatter" value="Serilog.Formatting.Compact.CompactJsonFormatter, Serilog.Formatting.Compact" />
+        <add key="serilog:write-to:File.path" value="%BASEDIR%\App_Data\Logs\UmbracoTraceLog.%MACHINENAME%..json" /> 
+        <add key="serilog:write-to:File.shared" value="true" />
+        <add key="serilog:write-to:File.restrictedToMinimumLevel" value="Debug" />
+        <add key="serilog:write-to:File.retainedFileCountLimit" value="" /> <!-- Number of log files to keep (or remove value to keep all files) -->
```

Comment:
can this be `string.Empty` ? (I have no idea .. may be right - just checking)

Reply:
Its either null/not set here or specify a number of files to keep (if set to days) it will be number of days files to keep for example

Reply:
I say keep them all for now, we can change at a later date if needed.